bottom-bar/src/main/java/com/roughike/bottombar/OverrideTabSelectionListener.java

```diff
@@ -18,12 +18,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-public interface TabOverrideListener {
+public interface OverrideTabSelectionListener {
```

Comment:
```setOverrideTabSelectionListener``` from a quick glance implies that we set a listener for overriding ```TabSelectionListener```, which certainly isn't the case. We're overriding tab selection alltogether, not tab selection listeners. :)

@yombunker ^^